src/common/schema-validator.ts

```diff
@@ -4,6 +4,7 @@ const {Validator} = require('jsonschema')
 import {ValidationError} from './errors'
 import {isValidAddress} from 'ripple-address-codec'
 import {isValidSecret} from './utils'
```

Comment:
Due to line 7, I think you'll need to remove this line.

Currently, the build fails with:
```
src/common/schema-validator.ts(6,9): error TS2300: Duplicate identifier 'isValidSecret'.
```